apps/web/components/OrderSummaryProductCard/OrderSummaryProductCard.vue

```diff
@@ -37,6 +37,17 @@
             </span>
           </li>
         </ul>
+        <ul class="text-xs font-normal leading-5 sm:typography-text-sm text-neutral-700">
+          <li v-for="(property, index) in orderGetters.getItemOrderProperties(orderItem)" :key="index">
```

Comment:
I understand that this would be a change on SDK side, but didn't we agree on converting the naming from "Item" to "Product" at that stage?

Reply:
@ksted, so the getter would be getProductOrderProperties? but It would still be `orderItem`? or would it be orderProduct? I think Item is a better term when talking about something that is part of an order. But we can discuss.